meetupHelpers.js

```diff
@@ -0,0 +1,52 @@
+const { joinArrayHumanReadable } = require('./utils');
+
+async function fetchNextEventForGroup (groupId) {
+  const fetch = await import('node-fetch');
```

Comment:
This import occurs foreach call to this function. Try loading in the library once like line 1.